pre-processing/primod/driver_coupling/ribamod.py

```diff
@@ -106,6 +106,27 @@ def write_exchanges(
                     save_flows=True,
                 )
                 destination = "river"
+
+                #  check on the bottom elevation and ribasim minimal subgrid level
+                minsglevel = (
+                    ribasim_model.basin.subgrid.df.groupby("subgrid_id")
+                    .min()[["subgrid_level"]]
+                    .to_numpy()
+                    .ravel()
+                )
+                # in active coupling, check subgrid levels versus modflow bottom elevation
+                if isinstance(self, RibaModActiveDriverCoupling):
+                    sugindex = mapping.dataframe["subgrid_index"]
+                    bndindex = mapping.dataframe["bound_index"]
+                    modflow_minlevel = package["bottom_elevation"].to_numpy().ravel()
+                    package_minlevel = modflow_minlevel[~np.isnan(modflow_minlevel)]
+                    ribamod_minlevel = np.empty_like(package_minlevel)
+                    ribamod_minlevel[:] = np.nan
+                    ribamod_minlevel[bndindex] = minsglevel[sugindex]
+                    if np.any(package_minlevel > ribamod_minlevel):
```

Comment:
Can't you just directly do (package_minlevel[river_index] > minsglevel[subgrid_index]).any() ?

Reply:
done